penny_university_frontend/src/components/follow-ups/FollowUpCard.tsx

```diff
@@ -19,22 +25,34 @@ type FollowUpCard = {
 }
 
 const FollowUpButtons = ({
-  editOnPress, saveOnPress, editMode, id,
-}: { editOnPress: () => void, saveOnPress: () => void, editMode: boolean, id: number }) => (editMode
-  ? <SaveButton className="align-self-start" type="Changes" onClick={saveOnPress} />
+  confirmDeleteOnPress, editOnPress, saveOnPress, editMode, id,
+}: {
+  confirmDeleteOnPress: () => void, editOnPress: () => void, saveOnPress: () => void, editMode: boolean, id: number
+}) => (editMode ? <SaveButton className="align-self-start" type="Changes" onClick={saveOnPress} />
   : (
     <Dropdown
       id={`followup-dropdown-${id}`}
       header="Options"
       testID={TestIDs.subMenu}
       options={[
-        <EditButton
-          className="align-self-start"
-          type="Follow Up"
+        <DropdownItem
+          className="btn btn-link"
           onClick={editOnPress}
           key={`edit-followup-${id}`}
-          color="link"
-        />,
+        >
+          <FontAwesomeIcon icon={faPen} />
+          {' '}
```

Comment:
I don't think we should keep carrying this pattern foward. I imagine we will hit a bug where the browser collapses white space. A better solution is to use css for spacing.

Reply:
I decided to replace the spaces with a `mr-2` utility class on the icon elements.